guides/source/getting_started.md

```diff
@@ -1917,12 +1917,12 @@ Our blog has <%= Article.public_count %> articles and counting!
 <%= link_to "New Article", new_article_path %>
 ```
 
-To finish up, we will add a select box to the forms, and let the user select the status when they create a new article or post a new comment. We can also specify the default status as `public`. In `app/views/articles/_form.html.erb`, we can add:
+To finish up, we will add a select box to the forms, and let the user select the status when they create a new article or post a new comment. Rather than repeat the literal list of statuses, we can make the code even DRYer and more expressive, by referencing `Visible::VALID_STATUSES`. We can also specify the default status as `public`. In `app/views/articles/_form.html.erb`, we can add:
```

Comment:
I don't think we need this change in the text. Let's just reuse the constant in the example, but we don't need to say "we can make the code even DRYer and more expressive, by referencing `Visible::VALID_STATUSES`"

Reply:
Not sure what the exact process around here is, sorry, couldn't find a handy doc on that.  Since you haven't modified and merged I ass-u-me the edit is up to me, so I'm gonna do that and re-resolve so it notifies you again.  I'm guessing you can merge it then, if there are no further objections.